transforms/universal/fdedup/ray/Dockerfile

```diff
@@ -13,24 +12,30 @@ COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
 RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
```

Comment:
should be getting the wheel instead of the source.

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/6cc18cd8eaba2fb12a31f49af52aba188a9f6ac4